docs/reference/synonyms/apis/delete-synonyms-set.asciidoc

```diff
@@ -0,0 +1,91 @@
+[[delete-synoyms-set]]
+=== Delete Synonyms Set
+
+beta::[]
+
+++++
+<titleabbrev>Delete Synonyms Set</titleabbrev>
+++++
+
+Deletes a Synonyms Set.
+
+[[delete-synonyms-set-request]]
+==== {api-request-title}
+
+`DELETE _synonyms/<synonyms_set>`
+
+[[delete-synonyms-set-prereqs]]
+==== {api-prereq-title}
+
+* Requires the `manage_search_synonyms` cluster privilege.
+* You can only delete a Synonyms Set that is not in use by any <<analysis-synonym-graph-tokenfilter,synonym graph token filters>> and <<analysis-synonym-tokenfilter,synonym token filters>> that are included in an analyzer. See <<delete-synonym-set-analyzer-requirements>> for more information.
+
+[[delete-synonyms-set-path-params]]
+==== {api-path-parms-title}
+
+`<synonyms_set>`::
+(Required, string)
+Synonyms set identifier to delete.
+
+
+[[delete-synonyms-set-response-codes]]
+==== {api-response-codes-title}
+
+`400`::
+The `synonyms_set` identifier was not provided, or the Synonyms Set can't be deleted because it does not meet the <<delete-synonym-set-analyzer-requirements>>.
+
+`404` (Missing resources)::
+No Synonyms Set with the identifier `synonyms_set` was found.
+
+[[delete-synonyms-set-example]]
+==== {api-examples-title}
+
+The following example deletes a Synonyms Set called `my-synonyms-set`:
+
+[source,console]
+----
+DELETE _synonyms/my-synonyms-set
+----
+
+
+[discrete]
+[[delete-synonym-set-analyzer-requirements]]
+==== Delete Synonyms Set analyzer requirements
```

Comment:
I am not sure if we need all these details and the whole [[delete-synonym-set-analyzer-requirements]] section.

Reply:
I think it's important to explain why users can't delete a synonyms set. It took me a while to understand it, and other members of the team as well.

Maybe it's a bit too much technical detail, but I would err on the side of providing more information that justifies the decision.